src/accelerate/utils/operations.py

```diff
@@ -298,6 +298,13 @@ def _gpu_gather_one(tensor):
         if not tensor.is_contiguous():
             tensor = tensor.contiguous()
 
+        # Check if `tensor` is not on CUDA
+        if state.device.type == "cuda" and tensor.device.type != "cuda":
+            raise RuntimeError(
+                "One or more of the tensors passed to `gather` was not on the GPU while the `Accelerator` is configured for CUDA. "
```

Comment:
I would have used "were", but not 100% sure:

```suggestion
                "One or more of the tensors passed to `gather` were not on the GPU while the `Accelerator` is configured for CUDA. "
```